packages/design-system/src/styles/components/_UsaBanner.scss

```diff
@@ -15,18 +15,24 @@
   --security-icon__color: #12890e; // spring-500
 
   background-color: var(--usa-banner__background-color);
-  color: var(--usa-banner-color);
```

Comment:
Here's a question... if USABanner doesn't define its color, it falls back to the color defined on `body` (`--global-color`). I did some playing around in Storybook and it seems like we'd never have a need to change this color to be different from the `body`, but I could be wrong.

If I'm not wrong, then could this rule be written as `color: var(--usa-banner-color, currentColor)` so banner defaults to inheriting the current color of its parent and allows for users to override this if that's not wanted?

Reply:
Are you saying it would allow us to feel secure in removing the `--usa-banner-color` from our tokens?

Reply:
Yeah, not defining it would result in it falling back like `currentColor`. Doing it this way would just be a little safer.

Reply:
Hmm, I actually think that the link color for medicare is wrong. If you look at [the production site](https://www.medicare.gov/), it uses the normal link color, and that seems more correct. Maybe we should get rid of both variables.